plugin.video.currenttime.tv/addon.py

```diff
@@ -0,0 +1,321 @@
+    '720p': 'http://rfe-lh.akamaihd.net/i/rfe_tvmc5@383630/index_0720_av-p.m3u8',
+    '540p': 'http://rfe-lh.akamaihd.net/i/rfe_tvmc5@383630/index_0540_av-p.m3u8',
+    '404p': 'http://rfe-lh.akamaihd.net/i/rfe_tvmc5@383630/index_0404_av-p.m3u8',
+    '288p': 'http://rfe-lh.akamaihd.net/i/rfe_tvmc5@383630/index_0288_av-p.m3u8'
+}
+video_url = {
+    '720p': '/index_2_av.m3u8',
+    '360p': '/index_1_av.m3u8',
+    '270p': '/index_0_av.m3u8'
+}
+main_menu = ([
+    [30003, 30004, 'lastvids+next', 	'folder',       '/z/17317.html'],   #Broadcasts
+    [30005, 30006, 'tvshows',			'folder',       ''],			    #TV Shows
+    [30007, 30008, 'lastvids+next', 	'folder',       '/z/17192.html'],   #All Videos
+    [30009, 30010, 'lastvids+next', 	'folder',       '/z/17226.html'],   #Daily Shoots
+    [30011, 30012, 'lastvids+next', 	'folder',       '/z/17318.html'],   #Reportages
+    [30013, 30014, 'lastvids+next', 	'folder',       '/z/17319.html']    #Interviews
+])
+tvshows = ([
+    [30031, 30032, 'lastvids+archive',  'olevski',	    '/z/20333.html'],
+    [30033, 30034, 'lastvids+archive',  'nveurope',     '/z/18657.html'],
+    [30035, 30036, 'lastvids+archive',  'nvasia', 	    '/z/17642.html'],
+    [30037, 30038, 'lastvids+archive',  'nvamerica',    '/z/20347.html'],
+    [30039, 30040, 'lastvids+archive',  'oba', 		    '/z/20366.html'],
+    [30041, 30042, 'lastvids+archive',  'itogi', 	    '/z/17499.html'],
+    [30043, 30044, 'lastvids+archive',  'week',		    '/z/17498.html'],
+    [30045, 30046, 'lastvids+archive',  'baltia', 	    '/z/20350.html'],
+    [30047, 30048, 'lastvids+archive',  'bisplan', 	    '/z/20354.html'],
+    [30049, 30050, 'lastvids+archive',  'unknownrus',   '/z/20331.html'],
+    [30051, 30052, 'lastvids+archive',  'guests', 	    '/z/20330.html']
+])
+default_view = {
+    'skin.confluence':
+        {'main_menu': '50', 'tvshows': '500', 'lastvids+next': '515', 'lastvids+archive': '515',
+         'allvids_archive': '51', 'video': '515'},
+    'skin.aeon.nox.5':
+        {'main_menu':'50', 'tvshows': '50', 'lastvids+next': '510', 'lastvids+archive': '510',
+         'allvids_archive': '50', 'video': '510'},
+    'skin.estuary':
+        {'main_menu': '55', 'tvshows': '500', 'lastvids+next': '502', 'lastvids+archive': '502',
+         'allvids_archive': '55', 'video': '502'}
+}
+
+NUM_OF_PARALLEL_REQ = 6    #queue size
+MAX_REQ_TRIES = 3
+MAX_ITEMS_TO_SHOW = 12
+video_pages_buffer = [None] * 30
+
+base_url = sys.argv[0]
+addon_handle = int(sys.argv[1])
+args = urlparse.parse_qs(sys.argv[2][1:])
+mode = args.get('mode', [None])[0]
+folder_url = args.get('folderurl', [None])[0]
+folder_title = args.get('title', [None])[0]
+folder_level = int(args.get('level', '0')[0])
+folder_name = args.get('name', [None])[0]
+
+site_url = 'http://www.currenttime.tv'
+addon_name = 'plugin.video.currenttime.tv'
+addon = xbmcaddon.Addon(addon_name)
+addon_icon = addon.getAddonInfo('icon')
+
+xbmcplugin.setContent(addon_handle, 'tvshows')  # !!!
```

Comment:
Is "videos" string a valid content type? It's not mentioned in the docs: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcplugin.html#gaa30572d1e5d9d589e1cd3bfc1e2318d6